src/ajax/check_hgvs_dialogue.php

```diff
@@ -128,15 +124,15 @@
     // We will fill in this fixed variant, close the dialogue,
     //  and perform a new call to this script by activating
     //  the onChange.
-    var oInput = $(\'input[name="' . $sFieldName . '"]\');
-    oInput.val("' . addslashes(lovd_fixHGVS($sVariant)) . '");
+    var oInput = $(\'input[name="' . htmlspecialchars($sFieldName) . '"]\');
+    oInput.val("' . htmlspecialchars(lovd_fixHGVS($_REQUEST['var'])) . '");
```

Comment:
`addslashes()` may have been enough (it escapes double quote also), although `htmlspecialchars()` is more thorough. I hope this won't cause issues with substitutions; was this checked? (just to be sure)